### ISSUE TITLE
Rename/document hrpc.Families to indicate that you can specify column qualifiers

### ISSUE THREAD

**Issue by tomasbanet** [tsuna/gohbase#191]
It is not immediately clear when creating a Scan request how one should specify the columns in a row that they are interested in retrieving.

After seeing some existing code written by others I saw that you can use the `tsuna/gohbase/hrpc/query.go:Families` option to specify the family as a key and the list of columns (qualifiers) in the family as a string slice in a map to get those families/columns in the result.

We should add some comments documenting what the Families map keys and values are supposed to represent. Alternatively rename the option to something like `FamiliesWithColumns` or `FamiliesWithQualifiers` to indicate that column selection is supported.

**Comment by dethi** [comment]
Right, I think the best would be to add documentation. Don't really think the other two functions would make sense.
Wanna contribue and make the change? :)

**Comment by tomasbanet** [comment]
Sure. I'll add it to the godoc
